internal/commands/root.go

```diff
@@ -205,6 +210,10 @@ func NewAstCLI(
 		chatCmd,
 	)
 
+	if wrappers.FeatureFlags[featureFlagsConstants.ByorEnabled] {
```

Comment:
I see you use the FF now,
few points to pay attention:
1. do you want to notify the user that this command is not available because the FF is off? if so, you need to add the command and hide him.
2. you have to add a test to check the situation with FF off
3. in the unit tests you have to set the FF=on - Otherwise the tests will fail (you can take reference from groups_test.go) 
4. you have to verify that the integration test will not fail when the FF is off